enhancements/authentication/allow-users-to-manage-their-own-tokens.md

```diff
@@ -0,0 +1,187 @@
+---
+title: neat-enhancement-idea
+authors:
+  - "@stlaz"
+reviewers:
+  - "@sttts"
+  - "@deads2k"
+  - "@marun"
+  - "@vareti"
+approvers:
+  - "@sttts"
+  - "@deads2k"
+creation-date: 2020-10-22
+last-updated: 2020-10-22
+status: implementable
+see-also:
+- [enhancement: Secure OAuth Resource Storage](https://github.com/openshift/enhancements/pull/323)
+replaces:
+superseded-by:
+
+---
+
+# Allow Users To Manage Their Own Tokens
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+While users are quite capable of retrieving access tokens to be able to
+work with OpenShift services, their capabilities to see the tokens they
```

Comment:
not sure what the first sentence means?

Reply:
users are able to create, use and delete tokens. They are not able to list their existing tokens.

Reply:
I've rewritten the summary completely to mention allowing to list specifically